testsupport/wait/host.go

```diff
@@ -965,7 +965,8 @@ func (a *HostAwaitility) printNotificationWaitCriterionDiffs(actual []toolchainv
 	} else {
 		buf.WriteString("failed to find notifications with matching criteria:\n")
 		buf.WriteString("actual:\n")
-		for _, obj := range actual {
+		for i := range actual {
+			obj := actual[i] // avoids the 'G601: Implicit memory aliasing in for loop. (gosec)' problem
 			y, _ := StringifyObject(&obj)
```

Comment:
is this really needed for this particular use-case - we should always get the correct values even with the original version...

Reply:
reverted in bc57e8a, using a `nolint:gosec` commend instead